module-2/calculating-games-of-chance/your-code/Untitled.ipynb

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
- Revisa los odds porque te está dando el mismo resultado que las probabilidades. Scipy tiene la función odds para calcular odds(), lo tenéis en la learning. Revisa la learning y después repasa la parte de los odds. 
- Intenta expresar siempre los odds en fracción probabilidad no evento: probabilidad evento
- Si el As puede valer 1 y 11 , dos ases pueden valer 2, 12 o 22 pero nunca 21. Un blackjack con dos ases no puede ser no?
- Esta parte no está bien explicada 
"Muestra numerica
Probabilidad de sacar 1 As
Prob As : 4/52
Probabilidad de convinacion de sacar dos As
52/ (52-2)!*2!
Probabilidad de sacar JQK: Teniendo en cuenta que existen 4 palos de cada 4*3= 12 y le sumamos los 4 As con los que cuenta la baraja con los que se llega alcanzar el BJ
16 / [52/ (52-2)!*2!]"